compiler/dyno/lib/parsing/ParserContextImpl.h

```diff
@@ -800,12 +810,22 @@ CommentsAndStmt ParserContext::buildFunctionDecl(YYLTYPE location,
       body = consumeToBlock(location, fp.body);
     }
 
-    auto f = Function::build(builder, this->convertLocation(location),
+    // Own the recorded identifier to clean it up, but grab its location.
+    owned<Identifier> identName = toOwned(fp.name);
+    assert(identName.get());
+    auto identNameLoc = builder->getLocation(identName.get());
+    assert(!identNameLoc.isEmpty());
+
+    // TODO: Right now this location is the start of the function, and
+    // it seems more natural for the location to be the symbol.
+    (void) location;
```

Comment:
Why is this statement here?

Reply:
I have this habit of voiding things that aren't really used to call them out for later removal. I removed it for you :)